rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -36,9 +36,10 @@ THE SOFTWARE.
  * \ingroup group_rocal_meta_data
  * \param [in] rocal_context rocal context
  * \param [in] source_path path to the folder that contains the dataset or metadata file
+ * \param file_list_path is the path to file list that contains the file names and its corresponding labels
```

Comment:
It is a text file right?
Can we mention that in the description

Reply:
I have seen file lists with .lst extension also so I just specified file list path here. Technically any file extension will work since we are using fread and reading the file paths-labels inside the file